jsk_perception/sample/sample_darknet_object_detection.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+  <arg name="launch_camera" default="true" />
```

Comment:
Could you please `launch_camera:=false` as default?
Most of the sample launch in jsk_perception is standalone.